src/app/lib/config/services/ukrainian.ts

```diff
@@ -45,16 +45,16 @@ const baseServiceConfig = {
   showAdPlaceholder: true,
   showRelatedTopics: true,
   podcastPromo: {
-    title: 'подкаст',
-    brandTitle: 'Що це було',
-    brandDescription: 'Головна історія тижня, яку пояснюють наші журналісти',
+    title: 'WhatsApp',
+    brandTitle: 'BBC Україна тепер у WhatsApp',
+    brandDescription: 'Головне про Україну та світ - у нашому WhatsApp-каналі',
     image: {
-      src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p09jrvvq.jpg',
-      alt: 'Що це було',
+      src: 'https://ichef.bbc.co.uk/images/ic/$recipe/p0kv11pb.png',
```

Comment:
Should this be 'bbci'?

Reply:
I think you're right, I've checked a few of the ohres and they're all bbci except the portuguese one which is bbc.

Reply:
fixed now